Automattic/wp-calypso
# Subscriptions management: Allow to unsubscribe from non-wpcom feeds

![uPvWCD.gif](https://github.com/Automattic/wp-calypso/assets/2019970/ba7a133c-87a5-4d30-80a3-8770de4a05a8)

Fixes https://github.com/Automattic/wp-calypso/issues/78631
Fixes https://github.com/Automattic/wp-calypso/issues/79419

## Proposed Changes

* Allow users to unsubscribe from their non-wpcom feed item subscriptions 
* After double-checking the unsubscribe endpoint for logged-in users, I found out that it requires either `sub_id` or `url` body parameter, while `sub_id` is preferred over `url`. Previously, we used to pass `blog_id` (unnecessarily) and `url`. I fix this, and so I do not pass the `blog_id` anymore, and instead `sub_id` is passed.

## Testing Instructions



* Go to http://calypso.localhost:3000/read/subscriptions
* Search for https://justinpot.com/feed
* Subscribe to the search result
* Remove the search input
* Unsubscribe from the "Justinpot" subscription
* Ensure it was successfully unsubscribed from

## Pre-merge Checklist
